packages/icons/src/library/audio.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const audio = (
+	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+		<Path d="M0,0h24v24H0V0z" fill="none" />
```

Comment:
This element will be invisible. It's usually the result of a "group" when the icon is being built, and they are useful when for example importing an SVG into Figma using drag and drop — in such a case it can help work as a group in the menu system as well.

However it's one reason why it might be nice to ponder an uatomated cleanup system.